src/aind_behavior_curriculum/tmp.py

```diff
@@ -0,0 +1,80 @@
+"""
+Work in Progress Curriculum
+"""
+
+from collections import defaultdict
+
+class Task:
+    """
+    Defined in exisiting PR
+    """
+    pass
+
+class Metric:
```

Comment:
I think this class should not wrap the database. This should only spec what the metrics oughta look like. Remember I may want to use Metrics without using the database.